### PR TITLE
Fixes Icebox making the Chaplain unable to replace their Altar

### PR DESCRIPTION
**Pull request by JohnFulpWillard**
## About The Pull Request

The Chaplain can only replace their altar of the gods if there's no other one that exists.
Icebox has two, the second one being in permabrig. This means that for a Chaplain to replace their altar, they have to also break the permabrig one.

Permabrig can't even use the altar, why do they have one?

## Why It's Good For The Game

Chaplain can now actually do what they are supposed to be intended to do via https://github.com/tgstation/tgstation/pull/62234

## Changelog

:cl:
fix: Icebox's Altar of the Gods was removed, letting Chaplains once again replace their own with a bible if it breaks.
/:cl:

### CODE REVIEW

**Review by san7890 — COMMENTED** [review]
Might be worth creating a toothless "fluff" variant of the altar for these sorts of mapping implementations in the future.

FYI: This looks fine, but we're still in the "freeze" state until I can get #66615, unless someone else thinks it's a critical issue that demands a speedmerge (then I'll manually merge it (which is annoying but I can still do that)).

**Comment by tralezab** [comment]
I don't like the idea of inert fluff copies of structures.

**Review by san7890 — APPROVED** [review]
Thank you!

File: _maps/map_files/IceBoxStation/IceBoxStation.dmm [hunk 1]
```diff
@@ -34887,8 +34887,8 @@
 /obj/machinery/atmospherics/pipe/smart/manifold4w/scrubbers/hidden/layer2,
 /obj/machinery/atmospherics/pipe/smart/manifold4w/supply/hidden/layer4,
 /obj/machinery/door/airlock/mining/glass{
-	name = "Quartermaster";
-	id_tag = "Quatermaster"
+	id_tag = "Quatermaster";
```
Comment (Fikou) [inline _maps/map_files/IceBoxStation/IceBoxStation.dmm:34890]: quatermaster

**Pull request merged**